ProjectSidewalk/SidewalkWebpage
# Addresses Issue #1019 . Adds code that allows Turkers to complete HITs "in the wild" on the main website.

See #1019 for more context and list of code additions.

1.  @misaugstad had submitted pull request #1009 which is yet to be approved. This pull request address the same issue but build on his code to provide additional functionality. @misaugstad should test this branch the same way as he would have #1009.
2. The additional bit of  testing to be done is as follows:
    - Check if the amt_assignment table has additional columns names "turker_id" and "confirmation_code".
    - Check if the mission_user table has an additional column called "paid".
    - Use the referrer "mturk" and provide hitId, assignmentId, and workerId parameters in the query string. For example: localhost:9000/?referrer=mturk&hitId=h9h9h9h&workerId=w9w9w9w9w9&assignmentId=a9a9a9a9
    - Check if you are logged in with the workerId as your username
    - Try auditing. At the end of your first mission you should see a button telling you to click it to generate a confirmation code.
    - Clicking the "Generate confirmation code" button should make the "Continue" button visible.
    - You should also be able to see the confirmation code in the amt_assignment table associated withe the correct hit_id, assignment_id, and turker_id.
    - You should not see the confirmation code or the button to generate it in the mission completion popups for the subsequent missions. 

Resolves #1019
Resolves #1037
